domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaycontextparameter/DisplayContextParameterValidator.java

```diff
@@ -6,9 +6,6 @@
 
 public interface DisplayContextParameterValidator {
 
-    Pattern NOT_ALLOWED_CHARACTERS_PATTERN_DATETIME = Pattern.compile("[VzOXxZ]");
-    Pattern NOT_ALLOWED_CHARACTERS_PATTERN_DATE = Pattern.compile("[ahKkHmsSAnNVzOXxZ]");
-
     DisplayContextParameterType getType();
 
     void validate(String parameterValue, String fieldType) throws Exception;
```

Comment:
I just looked into this implementation. both implementations are doing the same thing..
    public void validate(final String parameterValue, String fieldType) throws InvalidDateTimeFormatException {
        Pattern pattern = ((fieldType.equals(BASE_DATE)) ? NOT_ALLOWED_CHARACTERS_PATTERN_DATE : NOT_ALLOWED_CHARACTERS_PATTERN_DATETIME);
        dateTimeFormatParser.parseDateTimeFormat(parameterValue, pattern);
    }

except the values of NOT_ALLOWED_CHARACTERS_PATTERN_DATE  and NOT_ALLOWED_CHARACTERS_PATTERN_DATETIME are changed.

we can provide some methods in the interface to get the values from the runtime object, like:  DisplayContextParameterType getType();

Pattern getDateTimePattern()
Pattern getDatePattern();